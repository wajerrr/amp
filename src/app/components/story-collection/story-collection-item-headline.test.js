import React from 'react';
import renderer from 'react-test-renderer';
import ItemHeadline from './story-collection-item-headline';

describe('ItemHeadline', () => {
  it('should match snapshot without subheadline', () => {
    const tree = renderer
      .create(<ItemHeadline>headline</ItemHeadline>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should match snapshot with subheadline', () => {
    const tree = renderer
      .create(<ItemHeadline subheadline="subheadline">headline</ItemHeadline>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should match snapshot with subheadline and classNames', () => {
    const tree = renderer
      .create(
        <ItemHeadline
          className="class"
          subHeadlineClassName="class1"
          headlineClassName="class2"
          subheadline="subheadline"
        >
          headline
        </ItemHeadline>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
