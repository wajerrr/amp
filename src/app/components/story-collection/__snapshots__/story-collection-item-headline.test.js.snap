// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemHeadline should match snapshot with subheadline 1`] = `
<h3
  className="css-1sct6h1 "
>
  <span
    className="headline css-odjosd "
  >
    headline
  </span>
</h3>
`;

exports[`ItemHeadline should match snapshot with subheadline and classNames 1`] = `
<h3
  className="css-1sct6h1 class"
>
  <span
    className="headline css-odjosd class2"
  >
    headline
  </span>
</h3>
`;

exports[`ItemHeadline should match snapshot without subheadline 1`] = `
<h3
  className="css-1sct6h1 "
>
  <span
    className="headline css-odjosd "
  >
    headline
  </span>
</h3>
`;
